cob_default_robot_behavior/scripts/trigger_srvs_cob4-7.py

```diff
@@ -67,13 +67,28 @@
 from std_srvs.srv import Trigger, TriggerResponse
 
 def torso_front_cb(req):
-    sss.move_base_rel("base",[0,0,1.57],False)
-    sss.move("torso","front",True)
+    handle_arm_left = sss.move("arm_left","side", False)
+    handle_arm_right = sss.move("arm_right","side", False)
+    handle_arm_left.wait()
+    handle_arm_right.wait()
+    if handle_arm_left.get_error_code() == 0 and handle_arm_right.get_error_code() == 0:
+        sss.move_base_rel("base",[0,0,1.57],False)
+        sss.move("torso","front_down_full",True)
```

Comment:
Well, then this condition is not safe, too, as we do not check whether `sss.move("torso", ...)` was correct....we simply `return TriggerResponse(True, "")`